lib/wallet/http.js

```diff
@@ -468,16 +468,22 @@ class HTTP extends Server {
 
       const options = {
         rate: valid.u64('rate'),
+        blocks: valid.u32('blocks'),
         maxFee: valid.u64('maxFee'),
         selection: valid.str('selection'),
         smart: valid.bool('smart'),
         account: valid.str('account'),
+        sort: valid.bool('sort'),
         subtractFee: valid.bool('subtractFee'),
         subtractIndex: valid.i32('subtractIndex'),
         depth: valid.u32(['confirmations', 'depth']),
+        template: valid.bool('template', sign),
         outputs: []
       };
 
+      if (sign)
```

Comment:
I don't think this is necessary, and tests will pass without or without it. There is some duplicate work by `template` and `sign`, see [`mtx.template`](https://github.com/bcoin-org/bcoin/blob/master/lib/primitives/mtx.js#L1000-L1028), [`mtx.sign`](https://github.com/bcoin-org/bcoin/blob/master/lib/primitives/mtx.js#L1030-L1072), [`wallet.template`](https://github.com/bcoin-org/bcoin/blob/master/lib/wallet/wallet.js#L1698-L1710), and [`wallet.sign`](https://github.com/bcoin-org/bcoin/blob/master/lib/wallet/wallet.js#L1712-L1730).

Reply:
fixed.